compose-builder/add-security.yml

```diff
@@ -329,3 +329,38 @@ services:
       - security-bootstrapper
       - secretstore-setup
       - database
+
+  # this is to make sure this service is started after security-bootstrapper process is done
+  # because it needs to await Consul token to be available
+  app-service-rules:
+    entrypoint: ["/edgex-init/ready_to_run_wait_install.sh"]
+    command: "/app-service-configurable ${DEFAULT_EDGEX_RUN_CMD_PARMS}"
+    env_file:
+      - common-sec-stage-gate.env
+    environment:
+      # for this one doesn't need the secretstore
+      EDGEX_SECURITY_SECRET_STORE: "false"
+      # service key name is always lower cases
+      SERVICE_CONFIGACCESSTOKENFILE: /tmp/edgex/secrets/appservice-rules-engine/consul-token
```

Comment:
Please make update the ASC profiles for making it lower case there so we don't need these overrides. All the profiles will need updating.

Reply:
ok

Reply:
First see my comment on the PR with overrides for snap. I am suggesting a change in direction... :-)